src/typings/Options.ts

```diff
@@ -79,6 +88,21 @@ export const getOptions = async (): Promise<Options> => {
     const customTitle = getInput('custom-title');
     const coverageFile = getInput('coverage-file');
     const baseCoverageFile = getInput('base-coverage-file');
+    let prNumber: number | null = Number(
+        getInput('prnumber') || context?.payload?.pull_request?.number
+    );
+    let pull_request = null;
+
+    if (!Number.isNaN(prNumber)) {
+        const { data: pr } = await octokit.pulls.get({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            pull_number: prNumber,
+        });
+        pull_request = pr;
+    } else {
+        prNumber = null;
+    }
```

Comment:
I think you can do small optimization here. Something like:
```ts
let pull_request = context?.payload?.pull_request;

if(!pull_request && !Number.isNaN(prNumber))
```
Then you will avoid unnecessary requests when the context already has information about PR